shell/platform/embedder/embedder.h

```diff
@@ -2601,9 +2615,18 @@ FlutterEngineResult FlutterEngineAddView(FLUTTER_API_SYMBOL(FlutterEngine)
 //------------------------------------------------------------------------------
 /// @brief      Removes a view.
 ///
-///             This is an asynchronous operation. The view's resources must not
-///             be cleaned up until the |remove_view_callback| is invoked with
-///             a |removed| value of `true`.
+///             This operation notifies the engine to deallocate resources and
+///             inform Dart about the removal. Finally, it invokes
```

Comment:
I would remove this bit:

> This operation notifies the engine to deallocate resources and inform Dart about the removal.

In my opinion those are details unimportant to an embedder that are already implied by "Removes a view" above.